### PR TITLE
YJDH-176: Run Browser tests against test/staging environment

### PR DESCRIPTION
**Pull request by nenonja**
## Description :sparkles:
- Run browser tests against test and staging (every hour from 7 to 20 finnish time)
- Sends slack message from failed run to [#yjdh-alerts](https://app.slack.com/client/T02DHG0DR/C027C9KJWBX/thread/CHCV3KTHA-1635755060.025000) channel
- copy-pasted the implementation of workflow and slack messager from [events-helsinki-ui](https://github.com/City-of-Helsinki/events-helsinki-ui/pull/235)
- TODO: ask from backend api is the expected version deployed and run only if it is. Now there is a chance that the latest commit is not yet deployed. 

## Issues :bug:
https://helsinkisolutionoffice.atlassian.net/browse/YJDH-176

### CODE REVIEW

File: frontend/kesaseteli/employer/browser-tests/index-page/userActions.testcafe.ts [hunk 1]
```diff
@@ -40,20 +39,3 @@ test('can change to languages', async () => {
   await languageDropdown.actions.changeLanguage('en', 'fi');
   await headerUser.expectations.loginButtonIsTranslatedAs('fi');
 });
-
```
Comment (nenonja) [inline frontend/kesaseteli/employer/browser-tests/index-page/userActions.testcafe.ts:43]: test is obsolete after we implemented the idea of loading the ui language from draft application. There always might be a draft application left from previous run or manually testing which can be of any langauge.

File: .github/workflows/ks-staging-env-acceptance-tests.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    # - cron: "*/30 5-18 * * *"
+env:
+  FRONTEND_URL: https://yjdh-kesaseteli-ui-stg.apps.platta.hel.fi
+  BRANCH: main
+jobs:
+  acceptance-tests:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-latest
+    name: Acceptance tests for Test Environment
```
Comment (klemmari1) [inline .github/workflows/ks-staging-env-acceptance-tests.yml:15]: Acceptance tests for Staging Environment

Comment (nenonja) [reply .github/workflows/ks-staging-env-acceptance-tests.yml]: fixed

**Comment by klemmari1** [comment]
Should this be fixed in this PR as well?

https://github.com/City-of-Helsinki/yjdh/runs/4080369183?check_suite_focus=true

**Comment by nenonja** [comment]
> Should this be fixed in this PR as well?
> 
> https://github.com/City-of-Helsinki/yjdh/runs/4080369183?check_suite_focus=true

Something wrong with environment:
```
<body>
      <center>
      <h1>503 Service Temporarily Unavailable</h1>
      </center>
      <hr>
      <center>nginx</center>
```

I fixed the syntax error and pushed, let's see if this get sorted as well. The environment was ok now when I opened it:  https://ks-empl-431.test.kuva.hel.ninja

**Review by klemmari1 — APPROVED** [review]
Looks good to me!

**Pull request merged**